src/corona_hakab_model/consts.py

```diff
@@ -140,19 +130,21 @@ def from_file(param_path):
         with open(param_path, "rt") as read_file:
             data = read_file.read()
 
-        parameters = eval(data, {"__builtins__": None, "dist": dist, "rv_discrete": rv_discrete})
-        Consts.sanitize_parameters(parameters)
+        parameters = eval(
+            data,
+            {
+                "__builtins__": None,
+                "dist": dist,
+                "rv_discrete": rv_discrete,
+                "DetectionTest": DetectionTest,
+                "ConditionedPolicy": ConditionedPolicy,
+                "ConnectionTypes": ConnectionTypes,
+            },
```

Comment:
maybe this dictionary is too big to put in a call? place it in a separate variable?

Reply:
kk